postcss.config.js

```diff
@@ -1,17 +1,7 @@
-const purgecss = require("@fullhuman/postcss-purgecss")({
-  content: ["./src/**/*.html", "./src/**/*.svelte"],
-
-  whitelistPatterns: [/svelte-/],
-
-  defaultExtractor: (content) => content.match(/[A-Za-z0-9-_:/]+/g) || [],
-});
-
 const postCssImport = require("postcss-import");
 
 const tailwindcss = require("tailwindcss");
 
-const production = !process.env.ROLLUP_WATCH;
-
 module.exports = {
-  plugins: [postCssImport, tailwindcss, ...(production ? [purgecss] : [])],
+  plugins: [postCssImport, tailwindcss],
```

Comment:
To avoid a single-use variable you could instead write:
```
module.exports = {
  plugins: [
    require('postcss-import'),
    require('tailwindcss'),
  ],
}
```

Reply:
I've heard mention that purgeCSS won't purge classes when using svelte's [class directive](https://svelte.dev/tutorial/classes). In which case, purgeCSS needs a custom extractor to also find the `class:...` directives.

I've included one in the app-template branch here:
https://github.com/graph-paper-org/app-template/blob/tailwindcss/tailwind.config.js#L1-L15

Just note that the rest of the configuration is out of date, but that extractor may be useful.

Reply:
> To avoid a single-use variable you could instead write:

@robhudson Eslint doesn't like inline requires, so I added an eslint disable rule in my updated code to accommodate this change.

> I've heard mention that purgeCSS won't purge classes when using svelte's class directive. In which case, purgeCSS needs a custom extractor to also find the class:... directives.

Great insight! I just tested some class styling and Tailwind did not render at all. Thank you for the custom extractor, I'll definitely need it for my future Svelte projects.